.gitmodules

```diff
@@ -21,3 +21,9 @@
 [submodule "src/safeposix-rust"]
 	path = src/safeposix-rust
 	url = https://github.com/Lind-Project/safeposix-rust.git
+[submodule "tests/ipc_performance/total_runtime/scripts/native-rustposix"]
+	path = tests/ipc_performance/total_runtime/scripts/native-rustposix
+	url = https://github.com/Lind-Project/native-rustposix.git
+[submodule "tests/ipc_performance/read_write_vcs/scripts/native-rustposix"]
```

Comment:
is this supposed to be added twice?

Reply:
Yeah I think because it is for different tests.

Or I could just add this to the `tests/` folder

Reply:
Addressed this by moving the `native-rustposix` submodule to `tests/`